composer.json

```diff
@@ -1,7 +1,8 @@
 {
 	"name": "hm-base",
-	"license": "GPL V3+",
+	"license": "GPL-3.0-or-later",
 	"minimum-stability": "stable",
+	"type": "project",
```

Comment:
Do we want to follow the _default_ ordering of the properties, or not?

I read that Paul thinks this is too obsessive, but isn't this all about consistency, readability and even discoverability?
Sure, it's nothing that we should spent too much time over, every now and then again. That's exactly why I would like to set this once, now, and why not use the official documentation (that other tools are based on, or that they are using internally themselves, for example, the normalize package)...?

The order of the current properties would then be like so:

```json
	"name": "hm-base",
	"type": "project",
	"license": "GPL-3.0-or-later",
	"require": { ... },
	"require-dev": { ... },
	"minimum-stability": "dev",
	"prefer-stable": true,
	"repositories": [ ... ],
	"extra": { ... },
```

And I would add the following right before `"extra"`:

```json
	"config": {
		"platform": {
			"php": "7.0"   // Or whatever our required minimum PHP version is.
		}
	}
```

Also, this is still missing something like `"php": ">=7.0"` in the `"require"` property (as first entry).